### ISSUE TITLE
Idle-Mode and Screen-Off Detection enhancement

### ISSUE THREAD

**Issue by morbloe** [JPersson77/LGTVCompanion#125]
Currently there is no differentiation between User-Idle and Screen-Off Detection - if enabled this will led to TV being turned off. 
What I would like to have:
If User-Idle detected --> Black-Screen 
If Screen-OFF detected --> do nothing.

My motivation is, that windows starts to reorder the windows / applications since it detected that the TV is no longer available. (This is better handled in windows11, what I currently can't use)

In cases I use the companion-idle detection (by using Power-Tools keep awake) the LG companion will turn off the screen, but the TV keeps turned on. BUT This will also lead to any other attached monitor to keep being active, what I also do not want.
The goal is to use the windows idle detection which will lead to monitor signal lost and then the LG companion should blank the screen but keep the TV on.

Please consider this feature request, thank you very much.

**Comment by JPersson77** [comment]
Hi, thanks for the suggestion. If I understand correctly you would like the TV to be blanked instead of turned off when windows is shutting off the attached monitors. I can consider it as an option.

If you are by any chance using a 2-series LG you should probably enable "always ready" which keep the HDMI active so there is no shuffling etc

**Comment by morbloe** [comment]
Oh! Yes that helped with C2! Thanks. I also using Win11 now - so that solved it too. But maybe sill a valid enhancement! Thanks for your hint! Much appreciated!

**Comment by elduds** [comment]
For your black screen requirement, you could simply set a blank screen screensaver which does this

**Comment by morbloe** [comment]
yes, on my company-laptop I'm restricted to setup a screensaver. But beside that you are correct.

**Comment by JPersson77** [comment]
closing due to it being resolved. Feel free to reopen if needed
